package.json

```diff
@@ -1,46 +1,39 @@
 {
-  "name": "@replit/extensions",
-  "version": "1.1.0",
-  "description": "",
-  "exports": {
-    "./package.json": "./package.json",
-    ".": {
-      "types": "./dist/index.d.ts",
-      "import": {
-        "default": "./dist/index.js"
-      },
-      "default": "./dist/index.cjs",
-      "require": "./dist/index.js"
-    }
-  },
-  "types": "./dist/index.d.ts",
-  "type": "module",
-  "unpkg": "dist/index.global.js",
+  "name": "extensions",
+  "description": "Replit Extensions Client Monorepo",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "rm -rf dist && tsup src/index.ts  --sourcemap --dts --format esm,cjs,iife --global-name replit",
-    "lint": "npx prettier --write src/* && cd example && npx prettier --write .",
-    "lint:check": "npx prettier -l src/* && cd example && npx prettier -l .",
-    "type:check": "tsc --noEmit",
-    "dev": "npm run build && cd ./example && npm install && npm run dev",
-    "prepublishOnly": "npm run build",
-    "generate:docs": "npx typedoc src/index.ts --json docs/main.json --plugin ./util/signature-plugin/index.cjs"
+    "dev": "cd modules/example && pnpm dev",
```

Comment:
I think this script should also build the extensions package, or is that no longer required?